### PR TITLE
First feature

### PR DESCRIPTION
**Pull request by themonster2015**
In this PR, I: 

- Created an npm project with webpack.
- Wrote plain HTML markup with minimum styling to mimic the wireframe layout.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,10 +2,76 @@
...
+                id="score"
+                placeholder="Add your score"
+              />
+            </div>
+
+            <button type="submit" class="btn btn-outline-dark">Submit</button>
+          </form>
+        </div>
+      </div>
+    </div>
```
Comment (vichuge) [inline src/index.html:75]: - [ ] It looks like you create all the applications on `index.html`. On this script you must add something like `<div id="root"></div>`, then use javascript to take the created div and with js create all the application, in other words, is required don't use HTML code, in place, use JS code to make all the given template. 🐱 
- [ ] Don't forget to use ES6 best practices when you work on it. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Temmarie** [comment]
Hello  @themonster2015    :wave:

_This is an additional review as a result of the second opinion request._

You complained about the reviewer asking you to use JavaScript to generate the markup instead of plain HTML. The code reviewer followed guidelines for reviewing this milestone and since they were conflicting requested this change. As this was the only reason why your project was not approved, kindly request for a review and it will be approved immediately by the next code reviewer.
Keep rocking :rocket:!

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @themonster2015, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**